database/migrations/2020_11_04_144136_add_index_to_user_id_on_comments.php

```diff
@@ -17,7 +17,7 @@ class AddIndexToUserIdOnComments extends Migration
     public function up()
     {
         Schema::table('comments', function (Blueprint $table) {
-            $table->index('user_id');
+            $table->index(['user_id', DB::raw('created_at DESC'), DB::raw('id DESC')], 'created_at_desc');
```

Comment:
There's already an existing index with this name

Reply:
name it `user_comments` or something

Reply:
oh right, I got you wrong